Makefile

```diff
@@ -242,6 +242,19 @@ performance-profile-creator-tests: build-performance-profile-creator
 	@echo "Running Performance Profile Creator Tests"
 	hack/run-test.sh -t "test/e2e/performanceprofile/functests-performance-profile-creator" -p "--v -r --fail-fast --flake-attempts=2" -m "Running Functional Tests" -r "--junit-report=/tmp/artifacts"
 
+# Gather sysinfo binary for use in must-gather
+.PHONY: build-gather-sysinfo
+build-gather-sysinfo:
+	@echo "Building gather-sysinfo"
+	LDFLAGS="-s -w -X ${PACKAGE}/cmd/gather-sysinfo/version.Version=${REV} "; \
+	$(GO) build  -v $(LDFLAGS) -o $(OUT_DIR)/gather-sysinfo ./cmd/gather-sysinfo
+
+.PHONY: gather-sysinfo-tests
+gather-sysinfo-tests: build-gather-sysinfo
+	@echo "Running gather-sysinfo Tests"
+	$(GO) test  -v ./cmd/gather-sysinfo
```

Comment:
Nit: formatting double spaces.  I guess copied from `build-performance-profile-creator`

Reply:
Yeah, I assumed that was intentional :)